rmf_fleet_adapter/src/rmf_fleet_adapter/agv/FleetUpdateHandle.cpp

```diff
@@ -2538,6 +2538,13 @@ FleetUpdateHandle& FleetUpdateHandle::set_retreat_to_charger_interval(
   return *this;
 }
 
+//==============================================================================
+FleetUpdateHandle& FleetUpdateHandle::reassign_dispatched_tasks()
```

Comment:
Returning `FleetUpdateHandle&` is generally done in situations where users may want to chain a series of function calls. Usually that's when you're changing some settings for a class.

This function is triggering a distinct action, so I don't think it makes sense stylistically to chain any more function calls after it. Let's just return `void` from this function.